lib/gslab_model/test/make.sh

```diff
@@ -0,0 +1,22 @@
+export LOG=make.log
```

Comment:
- I think you should add a readme providing instructions on how to run these test (or add these instructions to the readme at the repo's root).
- Maybe we could use Python to run all the testing scripts as we do in [`gslab_matlab`](https://github.com/gslab-econ/gslab_matlab). I am not sure that the current approach will work on non-Unix platforms (let me know if you think it will).